Sources/SwiftDocC/Model/Rendering/Navigation Tree/RenderHierarchyTranslator.swift

```diff
@@ -54,32 +53,30 @@ struct RenderHierarchyTranslator {
             return nil
         }
         
-        let technologyReference = technologyPath[0]
-        var hierarchy = visitTechnology(technologyReference, omittingChapters: omittingChapters)
-        
+        let tutorialTableOfContentsReference = tutorialsPath[0]
+        var hierarchy = visitTutorialTableOfContents(tutorialTableOfContentsReference, omittingChapters: omittingChapters)
+
         hierarchy.paths = paths
             // Position the technology path as the canonical path for the node
             // in case it's curated multiple times under documentation symbols too.
-            .sorted(by: { (lhs, rhs) -> Bool in
-                return lhs == technologyPath
-            })
+            .sorted { lhs, _ in lhs == tutorialsPath }
             .map { $0.map { $0.absoluteString } }
         
-        return (hierarchy: .tutorials(hierarchy), technology: technologyReference)
+        return (hierarchy: .tutorials(hierarchy), tutorialTableOfContents: tutorialTableOfContentsReference)
     }
     
-    /// Returns the hierarchy under a given tutorials landing page.
-    /// - Parameter technologyReference: The reference to the tutorials landing page.
+    /// Returns the hierarchy under a given tutorials table-of-contents page.
+    /// - Parameter tutorialTableOfContentsReference: The reference to the tutorials table-of-contents page.
     /// - Parameter omittingChapters: If `true`, don't include chapters in the returned hierarchy.
-    /// - Returns: The hierarchy under the given landing page.
-    mutating func visitTechnology(_ technologyReference: ResolvedTopicReference, omittingChapters: Bool = false) -> RenderTutorialsHierarchy {
-        let technologyPath = urlGenerator.urlForReference(technologyReference, lowercased: true).path
-        collectedTopicReferences.insert(technologyReference)
+    /// - Returns: The hierarchy under the given tutorial table-of-contents page.
+    mutating func visitTutorialTableOfContents(_ tutorialTableOfContentsReference: ResolvedTopicReference, omittingChapters: Bool = false) -> RenderTutorialsHierarchy {
+        let technologyPath = urlGenerator.urlForReference(tutorialTableOfContentsReference, lowercased: true).path
```

Comment:
```suggestion
        let tutorialTableOfContentsPath = urlGenerator.urlForReference(tutorialTableOfContentsReference, lowercased: true).path
```

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1039/commits/44fece98cd31014ac38d962fbe0993358111d0bc